pkg/client/go.mod

```diff
@@ -0,0 +1,71 @@
+module github.com/mariadb-operator/mariadb-operator/pkg/client
+
+go 1.24.2
+
+replace github.com/mariadb-operator/mariadb-operator => ../../
+
+require (
+	github.com/mariadb-operator/mariadb-operator v0.0.0-00010101000000-000000000000
```

Comment:
Why this import? I believe this would bring all the dependencies from the root project here, something that defeats the purpose of having a dedicated module for the client. Prometheus operator client is not making this import:
https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/client/go.mod